tests/plugins/apps/remoteaccess/test_splashtop.py

```diff
@@ -0,0 +1,63 @@
+from __future__ import annotations
+
+from datetime import datetime, timezone
+from typing import TYPE_CHECKING
+from unittest.mock import patch
+
+from dissect.target.plugins.apps.remoteaccess.splashtop import SplashtopPlugin
+from tests._utils import absolute_path
+
+if TYPE_CHECKING:
+    from dissect.target.filesystem import VirtualFilesystem
+    from dissect.target.target import Target
+
+
+class StatMock:
```

Comment:
I meant something a little different, more like this:

```python
@pytest.fixture
def target_splashtop(target_win_users, fs_win):
    fs_win.map_file(
        "Program Files (x86)/Splashtop/Splashtop Remote/Server/log/SPLog.txt",
        absolute_path("_data/plugins/apps/remoteaccess/splashtop/SPLog.txt"),
    )
    entry = fs_win.get("Program Files (x86)/Splashtop/Splashtop Remote/Server/log/SPLog.txt")
    with patch.object(entry, "stat"):
        ...
        yield
```

And then use `target_splashtop` in your own tests.